src/DGtal/geometry/doc/moduleShrouds.dox

```diff
@@ -0,0 +1,189 @@
+
+/*
+ * Useful to avoid writing DGtal:: in front of every class.
+ */
+namespace DGtal {
+
+/**
+
+@page moduleShrouds Digital Surface Regularization by Shrouds algorithm
+
+@writers Jacques-Olivier Lachaud
+
+Part of the \ref packageGeometry.
+
+
+This part of the manual describes classes and functions related to the
+regularization of digital surfaces using the "Shrouds" algorithm, by
+Nielson et al \cite nielson2003shrouds. 
+
+On binary 3D images, the "Shrouds" algorithm builds a smooth surface
+that separates interior voxels from exterior voxels. It starts from
+the (closed) digital surface that separates interior voxels from
+exterior voxels. The 3D smooth surface reconstruction problem is
+transformed into a series of 2D smooth contour reconstruction
+problems, by slicing the 3D volume (and thus the digital surface) with
+axis-align planes. Each contour is regularized so as to minimize its
+squared curvature and the separation property is obtained by forcing
+each vertex to lie on its unit edge joining an interior voxel to an
+exterior voxel (i.e. the dual edge to the primal surfel). The 2D
+problems are intertwined since each vertex is shared by two
+slices. 
+
+@note This method is limited to closed surface reconstruction, in
+opposition with the method presented in \ref moduleRegularization.
+
+
```

Comment:
+disclaimer : it is not the author implem but what we interpreted to be a correct implem of the approach ?